lib/manageiq/messaging/client.rb

```diff
@@ -2,31 +2,105 @@ module ManageIQ
   module Messaging
     class Client
       def self.open(options)
+        # TODO: replace with a AMQP client
         StompClient.new(options)
       end
 
-      def publish_background_job(options)
-        BackgroundJob.push(self, options)
-      end
-
-      def subscribe_background_job(options)
-        BackgroundJob.subscribe(self, options)
+      # options
```

Comment:
fyi, a common format for these messages is http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md